documentation/tutorial.md

```diff
@@ -68,3 +75,762 @@ tutorial overlord    localhost/localhost  2.9.37   unsupported  23:20:53Z
+  status: completed
+  timing:
+    completed: 2022-12-02 14:53:30 +0000 UTC
+    enqueued: 2022-12-02 14:53:25 +0000 UTC
+    started: 2022-12-02 14:53:28 +0000 UTC
+```
+The admin password is under the result: `admin-password`. It should be different from your previous password.
+
+*Note when you change the admin password you will also need to update the admin password the in MongoDB URI; as the old password will no longer be valid.* Update the DB password used in the URI and update the URI:
+```shell
+export DB_PASSWORD=$(juju run-action mongodb/leader get-password --wait | grep admin-password|  awk '{print $2}')
+export URI=mongodb://$DB_USERNAME:$DB_PASSWORD@$HOST_IP/$DB_NAME?replicaSet=$REPL_SET_NAME
+```
+
+### Set the admin password
+You can change the admin password to a specific password by entering:
+```shell
+juju run-action mongodb/leader set-password password=<password> --wait
+```
+Running the command should output:
+```yaml
+unit-mongodb-0:
+  UnitId: mongodb/0
+  id: "4"
+  results:
+    admin-password: <password>
+  status: completed
+  timing:
+    completed: 2022-12-02 14:53:30 +0000 UTC
+    enqueued: 2022-12-02 14:53:25 +0000 UTC
+    started: 2022-12-02 14:53:28 +0000 UTC
+```
+The admin password under the result: `admin-password` should match whatever you passed in when you entered the command.
+
+*Note that when you change the admin password you will also need to update the admin password in the MongoDB URI, as the old password will no longer be valid.* To update the DB password used in the URI:
+```shell
+export DB_PASSWORD=$(juju run-action mongodb/leader get-password --wait | grep admin-password|  awk '{print $2}')
+export URI=mongodb://$DB_USERNAME:$DB_PASSWORD@$HOST_IP/$DB_NAME?replicaSet=$REPL_SET_NAME
+```
+
+## Relations
+<!---Juju 3.0 uses integrations; I haven’t been able to find the docs for 2.9 --->
+[Relations](https://juju.is/docs/sdk/integration), or what juju documentation describes as Integration, are the easiest way to create a user for MongoDB in Charmed MongoDB. Relations automatically create a username, password, and database for the desired user/application. As mentioned earlier in the [Access MongoDB](#access-mongodb) it is a better practice to connect to MongoDB via a specific user rather than the admin user.
+
+### Data Integrator Charm
+Before relating to a charmed application, we must deploy our charmed application. In this tutorial we will relate to the [Data Integrator Charm](https://github.com/canonical/data-integrator). This is a bare-bones charm that allows for central management of database users, providing support for different kind of data platforms (e.g. MongoDB, MySQL, PostgreSQL, Kafka, etc) with a consistent, opinionated and robust user experience. In order to deploy the Data Integrator Charm we must clone its source code from GitHub and create the charm executable ourselves: 
+
+<!--- note in the future replace this with juju deploy instead of from git --->
+```shell
```

Comment:
In the interim, should this be cloned into a separate folder?

Reply:
I think it is okay to keep it in this folder. In the future when someone does:
```
juju deploy data-integrator
```
it won't create a new folder since it grabs everything from Juju.

Reply:
Good to know! In that case, yeah it's probably fine to leave it as is, since it's all contained to the cloned folder